seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-kafka/src/main/java/org/apache/seatunnel/flink/kafka/source/KafkaTableStream.java

```diff
@@ -110,7 +111,12 @@ public void prepare(FlinkEnvironment env) {
         }
         String schemaContent = config.getString(SCHEMA);
         format = FormatType.from(config.getString(SOURCE_FORMAT).trim().toLowerCase());
-        schemaInfo = JsonUtils.parseArray(schemaContent);
+        try {
+            schemaInfo = JsonUtils.stringToJsonNode(schemaContent);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+        }
+        
```

Comment:
We need to determine what type it is, and then perform the corresponding conversion.

Reply:
FYI 
![image](https://user-images.githubusercontent.com/16631152/199214091-272debf8-b35d-47ec-a98e-a5e2bd55ad12.png)

Reply:
Sorry, Please ignore me, I didn't look closely at the method you used